src/components/WeeklySummariesReport/FormattedReport.jsx

```diff
@@ -121,18 +121,21 @@ const FormattedReport = ({ summaries, weekIndex, bioCanEdit }) => {
     );
   };
 
-  const getTotalValidWeeklySummaries = (summary, teamColor) => {
+  const getTotalValidWeeklySummaries = summary => {
+    const style = {
+      color: textColors[summary?.weeklySummaryOption] || textColors['Default'],
+    };
     if (summary.weeklySummariesCount === 8) {
```

Comment:
Why write two `getTotalValidWeeklySummaries` function?  They only differ based on the condition `summary.weeklySummariesCount === 8`, I think we can extract this part of the code:
`const getTotalValidWeeklySummaries = summary => {
  const style = {
    color: textColors[summary?.weeklySummaryOption] || textColors['Default'],
  };
  return (
    <p>
      <b style={style}>Total Valid Weekly Summaries:</b>{' '}
      {summary.weeklySummariesCount || 'No valid submissions yet!'}
    </p>
  );
};
`